subprojects/libero-distribution/README.md

```diff
@@ -37,6 +37,72 @@ installed distribution will be under `build/install`
   * config files can be specified as a parameter
   * searches in `<projectdir>/.libero.yml`
   * searches in `~/.libero/<project-name>/.libero.yml`
+  
+## Cleaning local artifacts
+
+Libero now supports cleaning artifacts from the local repo prior to building the project
+
+Artifacts can be cleaned by providing the `project.maven.repo.clean` property with a list of comma separated group ids
+Any artifact located at or below that group id depth will be deleted
+
+## Using an alternate local maven repository
+
+An alternate local maven repo can be configured by providing the `project.maven.repo.local` property with a path to use for a local repository
```

Comment:
My concern is if it is possible for the plugin to detect all dependencies.  For example, I am not sure if it will be able to calculate dependencies for feature files which is why we have the pom fix project.